src/borrow/migrations/031-maker-oracle-token-prices.sql

```diff
@@ -0,0 +1,14 @@
+create or replace function api.maker_oracle_token_prices(token character varying, date date)
+    returns TABLE(token character varying, price numeric, date date)
+    language sql
+as
+$$
+select
+        p.token,
+        p.price,
+        p.timestamp::date as date
+    from oracles.prices p
+    where p.token = $1 and p.price > 0
+    order by abs(p.timestamp::date - $2)
```

Comment:
ETH was running at 30ms, which is too slow.  We can use the index this way to bring it down to sub 1ms speed:

```suggestion
    where p.token = $1 and p.price > 0
    and p.timestamp between date - interval '5 days' and date + interval '5 days'
    order by abs(p.timestamp::date - $2)
```